docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
+          { pathname: '/x/react-data-grid/performance' },
         ],
       },
       {
-        pathname: '/x/react-data-grid/rows',
+        pathname: '/x/react-data-grid/advanced-features-group',
+        subheader: 'Advanced features',
         children: [
-          { pathname: '/x/react-data-grid/row-definition' },
-          { pathname: '/x/react-data-grid/row-updates' },
-          { pathname: '/x/react-data-grid/row-height' },
-          { pathname: '/x/react-data-grid/row-spanning', newFeature: true },
-          { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-ordering', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-pinning', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
+          { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
+          { pathname: '/x/react-data-grid/aggregation', plan: 'premium' },
+          { pathname: '/x/react-data-grid/pivoting', plan: 'premium', planned: true },
         ],
       },
-      { pathname: '/x/react-data-grid/editing' },
-      { pathname: '/x/react-data-grid/sorting' },
       {
-        pathname: '/x/react-data-grid/filtering-group',
-        title: 'Filtering',
+        pathname: '/x/react-data-grid/visual-customization-group',
+        subheader: 'Visual customization',
         children: [
-          { pathname: '/x/react-data-grid/filtering', title: 'Overview' },
-          { pathname: '/x/react-data-grid/filtering/customization' },
-          { pathname: '/x/react-data-grid/filtering/quick-filter' },
-          { pathname: '/x/react-data-grid/filtering/server-side', title: 'Server-side filter' },
-          { pathname: '/x/react-data-grid/filtering/multi-filters', plan: 'pro' },
-          {
-            pathname: '/x/react-data-grid/filtering/header-filters',
-            plan: 'pro',
-            newFeature: true,
-          },
-          { pathname: '/x/react-data-grid/filtering-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/style', title: 'Styling basics' },
+          { pathname: '/x/react-data-grid/style-recipes', title: 'Styling recipes' },
+          { pathname: '/x/react-data-grid/layout' },
+          { pathname: '/x/react-data-grid/overlays', title: 'Overlays' },
+          { pathname: '/x/react-data-grid/components', title: 'Custom subcomponents' },
         ],
       },
-      { pathname: '/x/react-data-grid/pagination' },
       {
-        pathname: '/x/react-data-grid/selection',
+        pathname: '/x/react-data-grid/interactivity-group',
+        subheader: 'Interactivity',
         children: [
-          { pathname: '/x/react-data-grid/row-selection' },
-          { pathname: '/x/react-data-grid/cell-selection', plan: 'premium', newFeature: true },
+          { pathname: '/x/react-data-grid/localization' },
```

Comment:
**Interactivity** seems like the wrong category for Localization, but I'm also not sure which would be better. Perhaps **Main features**?

Reply:
Maybe "Interactivity" should be removed altogether and these docs could be split between "Main" and "Advanced" features? None of the other products need this header so maybe it's not very useful here either.

Reply:
Yep, agree with that 👍